tests/misc/widgets/test_raw_fact_entry.py

```diff
@@ -0,0 +1,33 @@
+# -*- encoding: utf-8 -*-
+
+
+# This file is part of 'hamster-gtk'.
+#
+# 'hamster-gtk' is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# 'hamster-gtk' is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with 'hamster-gtk'.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Unittests for RawFactEntry."""
+
+from __future__ import absolute_import, unicode_literals
+
+from hamster_gtk.misc import widgets
```

Comment:
Maybe this could be changed to `from hamster_gtk.misc.widgets.raw_fact_entry import RawFactEntry` to match with the other file.